### DISCUSSION TITLE
取消遮罩颜色变化

### DISCUSSION THREAD

**Discussion by AceXa11** [Panuon/Panuon.UI.Silver#191] [Q&A]
弹窗的瞬间，遮罩颜色是动态变化的，环境配置较低，能否取消颜色变化，弹出后直接显示遮罩颜色

**Comment by Mochengvia** [comment]
pu没有提供这种方法
这种简单渐变的性能开销其实不高，建议检查一下，是否在创建新页面的过程中使用了耗时操作，导致卡主了UI线程？例如在构造函数中对 `ItemsSource` 赋值，或者执行了非异步的数据查询。
如果有，可以将这些操作延迟到页面Loaded之后执行。

**Comment by AceXa11** [comment]
感谢答复
主页面有个实时刷新图片的线程，ImageSource需要UIThread，我把刷新频率降低并且遮罩颜色变浅，显示效果可以，谢谢
